webapp/advantage/api.py

```diff
@@ -60,7 +60,11 @@ def get_accounts(self, email=""):
     def get_account_contracts(self, account_id: str):
         try:
             response = self._request(
-                method="get", path=f"v1/accounts/{account_id}/contracts"
+                method="get",
+                path=(
+                    f"v1/accounts/{account_id}/contracts"
+                    f"?productTags=ua&productTags=classic&productTags=pro"
```

Comment:
Does this means `("ua" || "classic" || "pro")`? It's a weird syntax I would've thought that query params were unique

Reply:
@ClementChaumel yes, it's works like an or. This is an array query param of the name `productTags`. Back-end will allow us to give the values comma separated in the future, but for now, this is what worked when I tested.

Reply:
It works. We also do it on https://ubuntu.com/security/cve?version=impish&status=released&version=hirsute&status=released

I was the proud dev to implement it.